data/scan-2026-05-08-13.md

```
@@ -0,0 +1,66 @@
+# Scan Report — 2026-05-08 (combined runs)
+
+## Run 1 — Earlier today (jobs 022–023)
+- **Searches run:** 6
+- **Jobs scoring 4.0+:** 2 → PDFs generated, notifications sent
+
+### 022 — Effectual | DevOps Engineer
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)